### DISCUSSION TITLE
繁体中文分支开发请求

### DISCUSSION THREAD

**Discussion by Elegantwolf** [NateScarlet/auto-derby#344] [Ideas]
不知是否有意向支持繁体中文？可以提供一些帮助。

**Comment by NateScarlet** [comment]
没兴趣 因为我不可能换服

[FAQ](https://github.com/NateScarlet/auto-derby/wiki/FAQ#%E4%B8%AD%E6%96%87%E7%89%88%E8%AF%86%E5%88%AB%E5%87%BA%E9%94%99) 如果更换语言会影响代码里写死的坐标 你可以自己fork

**Reply by Elegantwolf** [reply]
好的，了解

**Comment by TWei-1101** [comment]
> 不知是否有意向支持繁体中文？可以提供一些帮助。

@Elegantwolf Hi 請問有成功修改出繁中版本嗎?

**Reply by Elegantwolf** [reply]
还没，近期比较忙，忙到没有时间玩游戏，已经有两周没有打开游戏了😓
